sfge/src/main/java/com/salesforce/graph/source/supplier/AnnotationNamespaceAccessibleSupplier.java

```diff
@@ -0,0 +1,11 @@
+package com.salesforce.graph.source.supplier;
+
+import com.salesforce.graph.Schema;
+
+/** Supplier for non-test methods with a {@code @NamespaceAccessible} annotation. */
+public class AnnotationNamespaceAccessibleSupplier extends AbstractAnnotationSourceSupplier {
```

Comment:
Allows superclass method to mimic `PathEntryPointUtil#getNamespaceAccessibleMethods`.